scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
+  to calculate the Species Habitat Index (SHI) using Global Forest Watch data to measure 
+  changes in the habitat of the species
+inputs:
+  species:
+    label: species
+    description: Scientific name of the species
+    type: text
+    example: Saguinus oedipus
+  ref_system:
+    label: Projection
+    description: Reference system for the area of interest
+    type: int
+    example: 3116
+  elev_margin:
+    label: Elevation buffer
+    description: Magnitude of buffer to add to the elevation ranges of the species (usualy in meters), if no buffer then 0
```

Comment:
elevation buffer in meters to add to the reported species elevation range, default is zero.

Could it be a case to subtract from the elevation range?

Reply:
To substract a negative will be needed, and the code can receive it as it is, thus I added that explanation to the yml.
"Elevation buffer in meters to add (or substract) to the reported species elevation range. Defalut is zero. Positive values will increase the range in that magnitude and negative values will reduce the range in that magnitude."